pkg/sender/office365.go

```diff
@@ -78,6 +78,10 @@ func mapToGraphMail(email *email.SkylineEmail) (*graphusers.ItemSendMailPostRequ
 		contentType := graphmodels.HTML_BODYTYPE
 		body.SetContentType(&contentType)
 		body.SetContent(&email.HTML)
+	} else if email.IsMultiPartAlternative() {
```

Comment:
Could be merged with previous `if` block

Reply:
true, fixed